README.md

```diff
@@ -266,85 +266,55 @@ database, sets up the environment, configures python application to connect to A
 that works with a database. The application talks with the database via Acra, Acra **encrypts** the data before sending 
 it to a database, and decrypts the data when the app reads it from the database. Same it does transparently with tokenized data.
 
-### 2.1 Generate new zone
-
-```bash
-docker exec -it python-mysql_python_1 \
-  python3 extended_example_with_zone.py --host=acra-server --port=9393 --generate_zone
+### 3. MySQL Transparent Encryption
 
-$:
-zone_id: DDDDDDDDPfBoWiixeMTUuEOk
-zone public key in base64: b"UEC2\x00\x00\x00-\xbf\xc4\xd4\xa5\x02\x13ZTsg\x13\x88%R\xb5\x00\xc2\xbc\xe9\x9d\xa5\xa3i';\x7f)\xa8a\x9c\xdc\x9b\xc4\xba8\xb6\x04"
-```
+By default, Acra configured to work with MySQL database.
```

Comment:
let's mention, that by default in this demo. Because Acra in general by default configured to work with postgresql